Homework1/Task3_SumFromMaxToMin/Program.cs

```diff
@@ -0,0 +1,77 @@
+﻿/*Task 3. The application asks the user for the number of elements in a numeric array (at
+least two), and then in a loop -the elements themselves (of int type). After entering the
+elements, the application prints out the original array and outputs the sum of the array
+elements located between the smallest element in the array (the leftmost element if there
+are several) and the largest element (the rightmost element if there are several). Develop
+a console application that implements the specified functionality.
+Example: array[1, 3, 5, 1, 0, 3, 0, 1].The sum of the required elements = 5 + 1 + 0 = 6.
+Note 1: the correctness of the length of the array can be left unchecked.
+
+Note 2: The smallest and largest items are to be included in the amount.*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Homework_Task3
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            int[] array = { 8, 8, 0, 8, 5, 6 };
+            SumFromMaxToMinFinder(array);
+        }
+
+        static void SumFromMaxToMinFinder(int[] array)
+        {
+            var arrayAsList = array.ToList();
+
+            //Finding max value and max index
+            int maxValue = array.Max();
+            int maxIndex = arrayAsList.IndexOf(maxValue);
```

Comment:
It should be rightmost element in the array in a case of a few max. 

{ 8, 8, 0, **8**, 5, 6 }